ydb/public/lib/ydb_cli/commands/ydb_workload_import.cpp

```diff
@@ -46,61 +55,77 @@ int TWorkloadCommandImport::TUploadCommand::DoRun(NYdbWorkload::IWorkloadQueryGe
         Bar = MakeHolder<TProgressBar>(dataGen->GetSize());
         TVector<NThreading::TFuture<void>> sendings;
         for (ui32 t = 0; t < UploadParams.Threads; ++t) {
-            sendings.push_back(NThreading::Async([this, dataGen, &stop] () {
-                if (!ProcessDataGenerator(dataGen, stop)) {
-                    AtomicSet(stop, 1);
-                }
+            sendings.push_back(NThreading::Async([this, dataGen] () {
+                ProcessDataGenerator(dataGen);
             }, pool));
         }
         NThreading::WaitAll(sendings).Wait();
-        const bool wasErrors = AtomicGet(stop);
+        const bool wasErrors = AtomicGet(HasErrors);
         Cout << "Fill table " << dataGen->GetName() << "..."  << (wasErrors ? "Breaked" : "OK" ) << " " << Bar->GetCurProgress() << " / " << Bar->GetCapacity() << " (" << (Now() - start) << ")" << Endl;
```

Comment:
Breaked -> Failed
Also why three dots instead of a space? Single space would be enough imo